chef/automate
# #2762 guard add users button on team details view

### :nut_and_bolt: Description: What code changed, and why?

Before, the **Add Users** button on the team details page has always been shown.

With this change, it will only be shown of the user has got sufficient privileges to actually do that.

### Definition of done

- [X] hide the add users button if they don't have permission to add users to a team
- [x] if they happen to navigate to the add users page of a team, display this message on the page: It looks like you don't have permission to add users to this team. Reach out to your administrator or contact Chef Support for help. with Chef Support being an open in new tab link to our support page
- [X] changed/added tests pass

### :chains: Related Resources

#2762 

### :athletic_shoe: How to Build and Test the Change

- rebuild automate-ui in your preferred way
- sign in as admin, go to a team's details:

![Screenshot 2020-05-19 at 15 58 15](https://user-images.githubusercontent.com/870638/82339344-39d8d800-99ee-11ea-914a-881ed90ffaba.png)

- create a user and the policy mentioned in #2762, sign in as that user, go to the team's details page again, find the button gone:

![Screenshot 2020-05-19 at 15 59 20](https://user-images.githubusercontent.com/870638/82339436-5248f280-99ee-11ea-88e8-d39aa09d68ac.png)

- as `test` user, navigate to https://a2-dev.test/settings/teams/editors/add-users, notice:

![image](https://user-images.githubusercontent.com/870638/82428281-8ffb5900-9a8a-11ea-977d-d451711474ba.png)
